src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is needed since the new version of BeeJU doesn't provide logging config any more.

Reply:
it didn't provide this before either. should you need to add a log4j test dependency instead?

Reply:
Well, before it pulled in log4j transitively and by default this seems to have set things up to output to console. To achieve the same behaviour here I need to add this but I'm getting log4j2 transitively too, I'll make it a proper test dependency like you suggest.

Reply:
so this file can go now?

Reply:
No, it needs to be here if you want logging output in the tests, otherwise slf4j doesn't find any logging config and doesn't log anything:

> ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console.

Reply:
right ok